packages/product-components/package.json

```diff
@@ -19,6 +19,7 @@
     "dependencies": {
         "@suite-common/suite-constants": "workspace:*",
         "@suite-common/suite-types": "workspace:*",
+        "@suite-common/suite-utils": "workspace:*",
```

Comment:
this is imho antipattern. whatever is nested in `packages` must not import from whatever is nested in `suite-common` 
see more detailed explanation in https://www.notion.so/satoshilabs/Code-improvements-9-breaking-the-monolith-528bf80c34ba4a6bba3f335e60ff29e8